### ISSUE TITLE
User Preferences Not Saving on Test

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#673]
Steps:

1. User Preferences
2. Update Graph Visibility to Never
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/908d10cf-d314-473e-ab79-411e714a53a3)

3. Click Save - receive confirmation
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/bb205c90-d1a4-4ed4-a46f-50b9fc3d0e1c)

4. Go to Search History then back to User Preferences, the original defaults are restored
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/cddf1be8-5c43-4cfc-8eae-2223b656206e)

**Comment by dnsmith124** [comment]
Just tested and was able to recreate this, thankfully it looks like the preferences are applied properly but for some reason when the page is returned to _without_ reloading the page the preferences in the dropdowns don't display properly. 

So I did exactly what you described in the above, saw what you saw, but then when I ran a query the graph visibility was properly set to "never". 

When I refreshed the preferences page the dropdown for graph visibility showed "never" as it should. 

All of that said, this is a cosmetic issue in that the label on the preferences page is wrong when viewed without refreshing the page, but the actual setting itself is working correctly. Working on resolving that now.
